sdk/servicebus/Azure.Messaging.ServiceBus/tests/Core/ClientLibraryInformationTests.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
```

Comment:
This looks awfully familiar.    😛

Reply:
Must have been written by a very smart person :p